### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by peterpolman** [peterpolman/single-spa-aurelia-framework#3]
@jmezach Since you are the only one I known implementing this package you might have some additions or improvements for the readme. Feel free to submit a PR or let me know if it is sufficiently documented already.

Fyi: Also implemented support for `customProps()` to the adapter yesterday, which you might find interesting to include in your experiments.

**Comment by jmezach** [comment]
@peterpolman Great question. I was actually struggling a bit with the webpack.config. In my experiments I've created brand new Aurelia projects using the Aurelia CLI. The README suggests to extend the existing webpack.config rather than replacing the existing config, but I couldn't quite figure out how to do this, so I ended up making the necessary tweaks to the out-of-the-box webpack.config. That works, but obviously you can no longer run it in standalone mode now. Perhaps you could elaborate on how you actually extended the existing webpack.config. Did you use webpack-merge for that for example?

**Comment by jmezach** [comment]
Another thing I noticed after installing the new version is that I needed to change some types around which no longer matches the documentation. For example, the `getInstance` method now needs to return an `AureliaInstance` rather than an `Aurelia` type. Conversely the `configure` method now accepts a parameter `aurelia` of type `AureliaInstance`.

Perhaps it might even be better to not bother the user with the `AurelaInstance` type and only accept `Aurelia` types on the public API. Then internally we can do `as AureliaInstance` as needed (which I think is only in the `unmount` method).

**Comment by peterpolman** [comment]
Yeah I extended the Aurelia interface with some private properties that we use as publics now for teardown purposes. 

> Perhaps it might even be better to not bother the user with the AurelaInstance type and only accept Aurelia types on the public API. Then internally we can do as AureliaInstance as needed (which I think is only in the unmount method).

Your proposal makes a lot of sense though! I'll make that part of the next release or accept your PR if you're willing to submit one;)

**Comment by jmezach** [comment]
@peterpolman Sure, no problem. See #6 :).
